resources/views/edit_car.blade.php

```diff
@@ -75,6 +75,22 @@
               <input type="checkbox" class="form-check-input" style="padding: 0.7rem;" name="published" value="1" {{ old('published', $car->published ) ? 'checked' : '' }}>
             </div>
           </div>
+          <div class="form-group mb-3 row">
+            <label for="" class="form-label col-md-2 fw-bold text-md-end">Category:</label>
+            <div class="col-md-10">
+              <select name="cat_id" id="" class="form-control">
+                <option value="">Select Category</option>
+                @foreach($categories as $cat)
+                <option value="{{ $cat->id }}" @selected(old('cat_id', $car->cat_id) == $cat->id ? 'selected' : '')>
```

Comment:
use this
`@selected(old('cat_id', $car->cat_id) == $cat->id)`

or this 
`{{old('cat_id', $car->cat_id) == $cat->id ? 'selected' : ''}}`

not both😂